### PR TITLE
Dashboard UI

### PR DESCRIPTION
**Pull request by kshitij-hash**
Fixes: **#29 #28**

### Checklist

- [x] All acceptance criteria from the issue are met
- [x] Tested in latest Chrome/Firefox
- [x] Branch is up-to-date with the branch to be merged with, i.e., `main`
- [x] Code is cleaned up and formatted

### Summary of changes

- improved **pomodoro**UI
- added `dark mode` toggle
- updated `Task` Schema
- actions added: `getAllTasks`, `updateTask`
- added `TaskContext`

### Screenshots
Screenshot from 2024-12-05 17-53-26
Screenshot from 2024-12-05 17-53-37
Screenshot from 2024-12-05 17-53-42
Screenshot from 2024-12-05 17-54-32
Screenshot from 2024-12-05 17-54-37

### CODE REVIEW

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -13,7 +14,10 @@ export async function NavBar() {
         <Link href="/" className="font-bold">
```
Comment (piyushjha0409) [inline src/components/Navbar.tsx:14]: put Link and this user line at one div and give it styling

Comment (piyushjha0409) [reply src/components/Navbar.tsx:14]: from line no 13 to 16 it should be in one container

File: src/components/ui/tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+))
+TabsTrigger.displayName = TabsPrimitive.Trigger.displayName
+
+const TabsContent = React.forwardRef<
+  React.ElementRef<typeof TabsPrimitive.Content>,
+  React.ComponentPropsWithoutRef<typeof TabsPrimitive.Content>
+>(({ className, ...props }, ref) => (
+  <TabsPrimitive.Content
+    ref={ref}
+    className={cn(
```
Comment (piyushjha0409) [inline src/components/ui/tabs.tsx:46]: What is this can you explain me :smile: ? from line 46 to 52

**Review by piyushjha0409 — CHANGES_REQUESTED** [review]
Overall good, do the following changes and will approve changes after!

File: src/components/ui/tabs.tsx (hunk 2 above)
Comment (kshitij-hash) [reply src/components/ui/tabs.tsx:46]: this is shadcn component code sir
https://ui.shadcn.com/docs/components/tabs

**Review by piyushjha0409 — APPROVED** [review]
Looks good!

**Pull request merged**